src/main/resources/application.yml

```diff
@@ -57,6 +57,8 @@ prescriptionBucket: prescription-bucket
 gcpUrl: https://storage.googleapis.com/
 defaultImage: default-image.png
 token: userToken
+countryCodes:
+  - "+countryCodeOne"
```

Comment:
Hi @josdem 
Is this CountryCodeOne recognized to add + as a prefix?

Reply:
Hi @kaminuma country code is a String value, therefore it can contains any valid Java String, however we have a `user.html` template providing this value with a select field.

https://github.com/josdem/vetlog-spring-boot/blob/main/src/main/resources/templates/user/create.html

Not sure if I answer your question.